rock_paper_scissors.py

```diff
@@ -13,16 +13,16 @@
 
 
 if user_select == random_select:
```

Comment:
Написала зайві IF. якщо декілька умов - то краще використовавуати IF ELIF (це ELSE IF) ELSE констуркції.
Це оптимузіє умовні перевірки і кожен раз в тебе программа не буде прохожити по ВСІМ if

```suggestion
if user_select == random_select:
    print("game draw, sorry")
elif:
    user_select == "rock" and random_select == "paper"
    or user_select == "scissors" and random_select == "rock"
    or user_select == "paper" and random_select == "scissors":
    print("you lost, sorry")
else: 
    print("you win, congratulations!")
```

Reply:
Modified